frontend/lib/util/strings.ts

```diff
@@ -10,5 +10,5 @@ export function ellipsis(text: string, maxLength: number = 20): string {
 // returns undefined when non-numeric characters are encountered
 export function parseIntStrict(text: string): number | undefined {
   const num = parseInt(text, 10);
-  return !isNaN(num) && num.toString() === text ? num : undefined;
+  return !isNaN(num) && num.toString() === text.replace(/^0+/g, "") ? num : undefined;
```

Comment:
We should add a new function with a new name for this, instead of changing the behaviour of `parseIntStrict` to also strip leading zeroes.

Reply:
@praseodym Well what it does is it already strips the zeroes, because parseInt does this. The only thing I changed was the comparison to the original string, now removing the leading zeroes in the original string. Maybe I can change the name of the function to parsePollingStationNumber, to make it specific to this usage?

Reply:
I would create a new `parsePollingStationNumber` function that does this, and leave `parseIntStrict` as is. We also use `parseIntStrict` for parsing numeric IDs in URLs (`useNumericParam`) and there we do not want to strip leading zeroes.

Reply:
Updated in f834595efba02b693ee02046f91be1a6b9d7a55f